Objective-C/Tests/PredictiveQueryTest+CoreML.m

```diff
@@ -0,0 +1,361 @@
+//
+//  PredictiveQueryTest+CoreML.m
+//  CouchbaseLite
+//
+//  Copyright (c) 2018 Couchbase, Inc. All rights reserved.
+//
+//  Licensed under the Couchbase License Agreement (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  https://info.couchbase.com/rs/302-GJY-034/images/2017-10-30_License_Agreement.pdf
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "CBLTestCase.h"
+#import "CBLCoreMLPredictiveModel+Internal.h"
+
+API_AVAILABLE(macos(10.13), ios(11.0))
+@interface PredictiveQueryWithCoreMLTest : CBLTestCase
+
+@end
+
```

Comment:
extra line?
```suggestion


```

Reply:
OK

Reply:
Actually we don't add extra line here.

Reply:
before implementation is it only one line? but i could see CBLDocument, CBLFragment etc are all using two lines before the implementation starts.

Reply:
I don't think it's consistent. I will could make two lines here.